src/main/webapp/comment-tests.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>QUnit Tests</title>
+  <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-2.10.0.css">
+</head>
+
+<body>
+  <div id="qunit"></div>
+  <div id="qunit-fixture"></div>
+  <script src="https://code.jquery.com/qunit/qunit-2.10.0.js"></script>
+  <script src="https://unpkg.com/qunit-dom/dist/qunit-dom.js"></script>
+  <script src="https://sinonjs.org/releases/sinon-9.0.0.js"></script>
+  <script src="https://apis.google.com/js/api.js"></script>
+  <script src="scripts/script.js"></script>
+  <script type="module" src="scripts/tests/comment-post-test.js"></script>
+  <script defer type="module" src="scripts/tests/comment-post-test.js"></script>
+  <script defer type="module" src="scripts/tests/controller-auth-test.js"></script>
+
+  <!-- Async tests seem to have some conflicts with qunit-fixture.
+  Qunit may have restored the page before assertion happened. -->
```

Comment:
Does this mean the tests are flaky?

Reply:
this was Zoe's comment indicating that the html is here instead of qunit-fixture because tests were flaky otherwise, so they're not flaking anymore